src/components/Overview.tsx

```diff
@@ -0,0 +1,35 @@
+import { View, StyleSheet } from 'react-native'
+
+import { StyledText } from './StyledText'
+import { RefreshIcon } from './icons/Icons'
+import { colors } from '../config/themes/appThemes'
+
+interface OverviewProps {
+  updateTime: string
+}
+
+export const Overview: React.FC<OverviewProps> = (props) => {
+  const { updateTime } = props
+
+  return (
+    <View>
+      <StyledText style={styles.title} title2 neutral100>
+        Overview
+      </StyledText>
+
+      <StyledText style={styles.icono} body2 neutral80>
+        <RefreshIcon height={13} width={16} color={colors.neutral80}></RefreshIcon>
+        {updateTime}
+      </StyledText>
```

Comment:
Esto seria valido si en el Figma no hubiera un espacio entre el icono y el texto de 8. 
En este caso se debe usar la siguiente estructura y un estilo:
```suggestion
      <View style={styles.content}>
        <RefreshIcon height={13} width={16} color={colors.neutral80} />
        <StyledText body2 neutral80>
          {updateTime}
        </StyledText>
      </View>
```